coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -522,12 +506,12 @@ void copy_ivoc_vect_to_device(const IvocVect& from, IvocVect& to, bool vector_co
     /// if we need to copy IvocVect vector then newly alloated vector
     /// on the device is a new destination pointer
     if(vector_copy_needed) {
-        d_iv = (IvocVect*) cnrn_gpu_copyin((void*) &from, sizeof(IvocVect));
+        d_iv = cnrn_gpu_copyin(const_cast<IvocVect*>(&from));
```

Comment:
Wondering if the `..copyin` function should take a const pointer and return a non-const pointer, in which case this `const_cast` would not be needed. 🤔